config/kloubak2-subt-estop-lora.json

```diff
@@ -42,7 +42,8 @@
             "center_axle_distance": 0.348,
             "ad_center": 419.7,
             "ad_calibration_deg": 45,
-            "ad_range": -182.5
+            "ad_range": -182.5,
+            "max_join_angle": 1.19
```

Comment:
what are the units here?

Reply:
That are radians. I know, you prefer deg. Maybe we can use degrees here and then convert it to radians in the setup_global_const function. OK?

Reply:
we try to use degrees in "user facing" situations (like parameter in config file) since it is usually easier to visualize for the mere mortals :wink:

Reply:
Ok, see last commit.